lens/base/view-set.rkt

```diff
@@ -6,10 +6,11 @@
 (module+ test
   (require rackunit))
 
-(provide lens-view
-         lens-set
-         lens-view*
-         lens-set*)
+(provide
+ lens-view*
+ lens-set*
```

Comment:
re lens-view*: Oh duh, you're right.

re combinator: Yes that's exactly what I wanted but couldn't remember exactly how to do it. Will add that before merge.